System/PathFinder.cpp

```diff
@@ -1,9 +1,20 @@
 #include "PathFinder.h"
+#include "Material.h"
 #include "Scene.h"
 #include "SceneMan.h"
 
 namespace RTE {
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	PathNode::PathNode(const Vector& pos) : Pos(pos) {
+		const Material *outOfBounds = g_SceneMan.GetMaterialFromID(g_MaterialOutOfBounds);
+		for (int i = 0; i < c_MaxAdjacentNodeCount; i++) {
```

Comment:
Can probably use a range for loop here. But also, it's not a big deal.

Reply:
If you don't wanna bother, just resolve the comment and I'll live haha.
But it should probably be done.